tests/windows_test.go

```diff
@@ -255,6 +259,77 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 					`DNSDomain[\n\r\t ]+`+expectedSearchDomain+`[\n\r\t ]+`)
 			}, 360)
 		})
+
+		Context("VMI with bridge binding is created", func() {
+			var winVmiIp string
+			var winPodIp string
+
+			BeforeEach(func() {
+				windowsVMI.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					{
+						Name: "default",
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							Bridge: &v1.InterfaceBridge{},
+						},
+					},
+				}
+
+				By("Starting Windows VirtualMachineInstance with bridge binding")
+				windowsVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(windowsVMI)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
+
+				cli = winrnLoginCommand(virtClient, windowsVMI)
+			})
+
+			It("should have ip value equal to matching pod ip", func() {
+				windowsVMI, err = virtClient.VirtualMachineInstance(windowsVMI.Namespace).Get(windowsVMI.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() int {
+					return len(windowsVMI.Status.Interfaces)
+				}, time.Minute*1, time.Second*15).Should(BeNumerically("==", 1))
+
+				winVmiIp = windowsVMI.Status.Interfaces[0].IP
+				winVmiPod := tests.GetRunningPodByVirtualMachineInstance(windowsVMI, windowsVMI.Namespace)
+				winPodIp = winVmiPod.Status.PodIP
+
+				Expect(winVmiIp).To(Equal(winPodIp))
+			})
+
+			It("should recognize other vmis via ping command", func() {
+
+				By("creating helper vmi for ping test")
+				testVMI := libvmi.NewCirros()
+
+				By("creating and starting test VMI")
+				testVMI, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(testVMI)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStartWithTimeout(testVMI, 360)
+
+				testVMI = tests.WaitUntilVMIReady(testVMI, console.LoginToCirros)
+
+				Eventually(func() int {
+					return len(testVMI.Status.Interfaces)
+				}, time.Minute*1, time.Second*15).Should(BeNumerically("==", 1))
+
+				testVmiIp := testVMI.Status.Interfaces[0].IP
+
+				command := append(cli, fmt.Sprintf("ping %s", testVmiIp))
```

Comment:
I do not think we need a VMI to run pings to. This is the pod network if I understand correctly, therefore it is enough to ping the virt-handler (and we use this trick in several other tests).
If that is not enough (in case it is not the pod network for example), raising a pod is the second best option. Raising a VMI is expensive.